changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.3.10
+
+Creates new Debian package `securedrop-keyring` for managing the SecureDrop
+Release Signing Key. Rotates the signing key currently in use by setting
+a dependency on the other Debian packages, so that currently running deployments
+will have their apt keyrings updated via automatic nightly updates.
```

Comment:
@conorsch Agreed! I simply think it would be clearer if the changelog was written to express those two things separately, e.g.

"Creates new Debian package `securedrop-keyring` for managing the SecureDrop Release Signing Key. Adds the new keyring package as a dependency to the other SecureDrop packages so that currently running deployments will have their apt keyrings updated via automatic nightly updates.

In 0.3.10, `securedrop-keyring` rotates the code signing key used for SecureDrop. Since the keyring package will be installed automatically as a dependency of the other SecureDrop packages, this rotation will happen automatically and does not require admin intervention."

But this is a minor point. If you prefer the original wording, leave it and we'll ship it.